Makefile

```diff
@@ -0,0 +1,20 @@
+NAMESPACE=test-app
```

Comment:
Should we keep the namespace the same name as the repo?

Reply:
Sounds good. Will make the change.